sdk/core/core-rest-pipeline/test/bearerTokenAuthenticationPolicy.spec.ts

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { assert } from "chai";
-import * as sinon from "sinon";
-import { AccessToken, TokenCredential } from "@azure/core-auth";
+import { assert, describe, it, beforeEach, afterEach, vi, expect } from "vitest";
+import type { AccessToken, TokenCredential } from "@azure/core-auth";
 import {
-  PipelinePolicy,
-  PipelineResponse,
-  SendRequest,
+  type PipelinePolicy,
```

Comment:
I wonder if we should separate mechanical changes like this and do one PR-wide `verbatimModuleSyntax` change...

Maybe we can automate that via a codemod as well. I can try it!

Reply:
Yeah I was looking at using [ts-fix](https://github.com/microsoft/ts-fix) for this!

Reply:
I heard that this typescript-eslint rule can also be helpful

Reply:
https://typescript-eslint.io/rules/consistent-type-imports/

Reply:
Good find! 👍 We could try the eslint rule with a NO_CI type of build ?